src/randoop/ExceptionCheck.java

```diff
@@ -0,0 +1,157 @@
+package randoop;
+
+import java.util.Objects;
+
+/**
+ * An {@code ExceptionCheck} is used to indicate that an exception is expected
+ * at a particular statement in a sequence. Depending on command-line arguments
+ * to Randoop, an instance may be either a {@link ExpectedExceptionCheck} or 
+ * {@link EmptyExceptionCheck}. Both will catch the exception if it occurs, but
```

Comment:
"Both will catch" makes it sound like the ExceptionCheck catches the exception.  I think you mean that code in the generated test will catch the exception, and that would be clearer to say.  Reading this comment, I'm not clear on whether that code is generated by ExceptionCheck or elsewhere, and clarifying that here would be helpful to people who want to understand the code.